### PR TITLE
Prac 10 Feedback

### PR DESCRIPTION
**Pull request by ivy-naw-mth**
@KuiMengcen
@ChristineYamin
@Kailing09

Would you guys mind helping me check my codes? 
Much appreciated and thanks in advance! ^^

### CODE REVIEW

**Comment by KuiMengcen** [comment]
good code

File: prac_10/README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+3. Regarding practical tasks overall, what would you change if you were in charge of the subject? 
+
+    * If I were in charge of this subject, I'd introduce group presentations to provide valuable experiences and diversity of the projects to showcase Python's applications across various domains. Exploring Kivy further could enhance our understanding of its capabilities, while reflecting on common mistakes weekly can help all students refine their work based on feedback. While keeping practical tasks mostly unchanged, I'd suggest adding outputs to clarify topics and include solutions to aid understanding and problem-solving.
+
+
+# Code Reviews
+
+1. [Prac07 - Code Review] (https://github.com/Quinn368/cp1404practicals/pull/4#issuecomment-2028794426) Review about "while True" is not good for standard coding format.
+2. [Prac 06 - Code Review] (https://github.com/Anastasia-Eremenko/CP1404_ana/pull/2#event-12224370340) Review about "Dictionaries and their usages".
```
Comment (ChristineYamin) [inline prac_10/README.md:19]: Clear and complete ReadMe.md

File: prac_10/app.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+@app.route('/c/<fahrenheit>')
+def convert_to_celsius(fahrenheit):
+    celsius = 5 / 9 * (float(fahrenheit) - 32)
+    return str(celsius)
+
+
+@app.route('/f/<celsius>')
+def convert_to_fahrenheit(celsius):
```
Comment (ChristineYamin) [inline prac_10/app.py]: I think you need to write "celsius= 22(as an example)" .

Comment (ivy-naw-mth) [reply prac_10/app.py]: Got it, I will fix that, thanks! ^^

**Pull request merged**